clients/object_store_client.go

```diff
@@ -184,8 +184,8 @@ func newExponentialBackOffExecutor() *backoff.ExponentialBackOff {
 	backOff := backoff.NewExponentialBackOff()
 	backOff.InitialInterval = 200 * time.Millisecond
 	backOff.MaxInterval = maxRetryInterval
-	backOff.MaxElapsedTime = 0 // don't impose a timeout as part of the retries
-
+	backOff.MaxElapsedTime = 0 // don't impose a timeout as part of the retries=
```

Comment:
Trailing `=`? :)

Reply:
oh yeah i think i was coding with the box running and my mac was toasting lol